docs/README_RECON.md

```diff
@@ -87,7 +98,28 @@ class Table:
     transformations: list[Transformation] | None = None
     thresholds: list[Thresholds] | None = None
     filters: Filters | None = None
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+{
+  "source_name": "product",
+  "target_name": "product",
+  "join_columns": ["p_id"],
```

Comment:
We can provide an example JSON file with all the details, at the end. (maybe collapsed one)

Reply:
Fixed